modules/View.js

```diff
@@ -742,7 +743,7 @@
 
           if (prepareFieldValue && _.isFunction(prepareFieldValue.toJSON)) {
             prepareFieldsContext[prepareFieldName] = prepareFieldValue.toJSON();
-          } else if (!prepareFieldIsSimpleString && prepareFieldValueIsDefinedOnView) {
+          } else if (!prepareFieldIsSimpleString || prepareFieldValueIsDefinedOnView) {
```

Comment:
random bug this logic was backwards and it wouldn't work if it was a string reference to a view field value.